drivers/spi/spi_rpi_pico_pio.c

```diff
@@ -365,6 +381,11 @@ static int spi_pico_pio_configure(const struct spi_pico_pio_config *dev_cfg,
 			wrap_target = RPI_PICO_PIO_GET_WRAP_TARGET(spi_mode_1_1);
 			wrap = RPI_PICO_PIO_GET_WRAP(spi_mode_1_1);
 			cycles = SPI_MODE_1_1_CYCLES;
+		} else if ((cpol == 0) && (cpha == 1)) {
+			program = RPI_PICO_PIO_GET_PROGRAM(spi_mode_0_1);
+			wrap_target = RPI_PICO_PIO_GET_WRAP_TARGET(spi_mode_0_1);
+			wrap = RPI_PICO_PIO_GET_WRAP(spi_mode_0_1);
+			cycles = SPI_MODE_0_1_CYCLES;
 		} else {
 			LOG_ERR("Not supported:  cpol=%d, cpha=%d\n", cpol, cpha);
```

Comment:
unrelated to your change but no `\n` in LOG_ statement, might be worth cleaning up in a separate commit if you feel like it

Reply:
Done.